### DISCUSSION TITLE
caratteri troppo piccoli

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#368] [Richieste di aiuto]
Buongiorno,nuovamente i genitori hanno ripreso con il lamentarsi del fatto che i caratteri sono troppo piccoli,soprattutto quando vanno a vedere la situazione voti ,dicono che le celle con il voto  risultano troppo piccole e poco evidenti.Si può fare qualcosa?Gli ho spiegato di effettuare lo zoom dai tre puntini in alto a destra , ma a quanto pare loro pretendono già di default un carattere più grande ed evidente

**Comment by trinko** [comment]
La segno come richiesta, ma modifiche all'interfaccia utente non sono previste nel breve periodo. 

Se si conosce l'uso degli stili CSS si possono autonomamente modificare le impostazioni di visualizzazione.
